src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -1,9 +1,11 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import getProp from 'lodash/get';
+import fuzzysearch from 'fuzzysearch';
 import elementsMessages from 'box-elements-messages'; // eslint-disable-line
 import intlLocaleData from 'react-intl-locale-data'; // eslint-disable-line
 import Internationalize from 'box-ui-elements/es/elements/common/Internationalize';
+import SearchBar from 'box-ui-elements/es/elements/common/header';
```

Comment:
It looks like the `Header` component is doing a lot of stuff we don't need (isSmall, logo, disablingbased on view prop). I think we should be wary about importing individual pieces from the `/es/elements` directory. Should we create our own simple input, instead?